tests/unit/resources/help.txt

```diff
@@ -29,7 +29,9 @@ Commands:
     server stop
     server uninstall
     server upgrade
+    slider install
```

Comment:
Same as last time; this is going to be updated once mode-switching goes in for both slider_install/uninstall and install/uninstall